include/multipass/ssh/sftp_dir_iterator.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2019-2022 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_SFTP_DIR_ITERATOR_H
+#define MULTIPASS_SFTP_DIR_ITERATOR_H
+
+#include <filesystem>
+#include <functional>
+#include <libssh/sftp.h>
```

Comment:
It's preferred have all of the standard lib headers defined together with `#include <libssh/sftp.h>` defined first with a blank line below it before the standard lib block of headers.